java/cli/src/main/java/org/ray/cli/CommandSubmit.java

```diff
@@ -0,0 +1,27 @@
+package org.ray.cli;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+/**
+ * Arguments for command submit
+ */
+@Parameters(separators = "= ", commandDescription = "submit a job to ray cluster")
+public class CommandSubmit {
+
+  // must exist.
+  @Parameter(names = "--package", description = "java jar package zip file")
+  public String packageZip ;
+
+  // must exist
+  @Parameter(names = "--class", description = "java class name")
```

Comment:
Good main class input. May we should also pay attention to general case that user could has some input args in the  main class. Some params like "--args " should be supported.

Reply:
Good point, thanks.